homeassistant/components/climate/daikin.py

```diff
@@ -22,7 +22,7 @@
     ATTR_TEMPERATURE, CONF_HOST, CONF_NAME, TEMP_CELSIUS)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['pydaikin==0.8']
+REQUIREMENTS = ['pydaikin==0.9']
```

Comment:
The platform shouldn't have the requirements when there is a component. This should be removed.

Reply:
There is always something to pick on 😏